src/library.js

```diff
@@ -4647,6 +4647,21 @@ LibraryManager.library = {
     else return lengthBytesUTF8(str);
   },
 
+  emscripten_wasm_return_address__deps: ['emscripten_get_callstack_js'],
+  emscripten_wasm_return_address: function(level) {
```

Comment:
It might be nice to mention the addition of this function in the changelog to surface it to any other interested users.